google_chat_ros/src/google_chat_ros/google_chat.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import print_function
+
+from apiclient.discovery import build
+from googleapiclient.errors import HttpError
```

Comment:
Fix python script according to linter.

For example, flake8 says:
```
$ flake8 google_chat.py 
google_chat.py:4:1: F401 'googleapiclient.errors.HttpError' imported but unused
google_chat.py:6:1: F401 'json' imported but unused
google_chat.py:9:1: E302 expected 2 blank lines, found 1
google_chat.py:18:80: E501 line too long (137 > 79 characters)
google_chat.py:21:80: E501 line too long (110 > 79 characters)
google_chat.py:22:80: E501 line too long (83 > 79 characters)
google_chat.py:26:80: E501 line too long (170 > 79 characters)
google_chat.py:26:81: E231 missing whitespace after ':'
google_chat.py:26:98: E261 at least two spaces before inline comment
```

Reply:
Fixed without some comments